core/src/test/java/com/google/errorprone/bugpatterns/time/ZoneIdOfZTest.java

```diff
@@ -63,6 +76,18 @@ public void zoneIdOfLowerCaseZ() {
         .doTest();
   }
 
+  @Test
+  public void zoneIdOfLowerCaseUtc() {
+    helper
+        .addSourceLines(
+            "TestClass.java",
+            "import java.time.ZoneId;",
+            "public class TestClass {",
+            "  private static final ZoneId UTC = ZoneId.of(\"utc\");",
```

Comment:
Not a huge deal, but http://errorprone.info/bugpattern/InvalidZoneId should prevent this from compiling in the first place :-)